### ISSUE TITLE
`docpad --profile` gives an Error

### ISSUE THREAD

**Issue by Zearin** [docpad/docpad#937]
Doesn’t matter what the command after `--profile` is—I get the same error, regardless.

```
$ docpad --profile info
Loading profiling tool: webkit-devtools-agent
/Users/tony/<PROJECT>/node_modules/docpad/out/lib/docpad.js:39
    lazyRequire('webkit-devtools-agent', {
    ^
TypeError: undefined is not a function
    at Object.<anonymous> (/Users/tony/<PROJECT>/node_modules/docpad/out/lib/docpad.js:39:5)
    at Object.<anonymous> (/Users/tony/<PROJECT>/node_modules/docpad/out/lib/docpad.js:4594:4)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at startDocPad (/Users/tony/<PROJECT>/node_modules/docpad/out/bin/docpad.js:27:14)
    at checkDocPad (/Users/tony/<PROJECT>/node_modules/docpad/out/bin/docpad.js:17:14)
An error occured within the child DocPad instance: exited with a non-zero status code
```

Using:
* Docpad v6.73.5
* NPM 2.6.0
* Node 0.12.0
* OS X 10.10 (“Yosemite”)

**Comment by Zearin** [comment]
Forgot to mention my Docpad plugins:

* browserifybundles
* cson
* growl
* livereload

**Comment by balupton** [comment]
Identified the issue. I can do a quickfix today, or a more long-term fix next week - which will you prefer?

**Comment by Zearin** [comment]
Fix it for the long haul. 

Thanks!

**Comment by fengtingzhang** [comment]
What was the issue? I am on version 6.69.2 and I am getting the same error.

**Comment by balupton** [comment]
Fixed in v6.74.0

**Comment by Zearin** [comment]
Thanks @balupton !
